pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.sagebionetworks</groupId>
     <artifactId>Bridge-Reporter-Scheduler</artifactId>
-    <version>1.0</version>
+    <version>1.1</version>
```

Comment:
This is unnecessary. Bridge-Reporter-Scheduler is deployed directly to Lambda, not built into Maven.